Phonebook.Frontend/src/app/app.component.ts

```diff
@@ -95,8 +99,19 @@ export class AppComponent implements OnInit {
     //   });
     // }
 
+    //if skip_cookies is set, dont show dialogs
+    if (this.skipDialog) {
+      this.store.dispatch(new SetDisplayedNotificationVersion(DisplayNotificationDialog.version));
+    }
+    //subscribe on query param changes, if changed open snackbar
+    this.activatedRoute.queryParamMap.pipe(untilComponentDestroyed(this)).subscribe(queryParamMap => {
+      if (queryParamMap.get('skip_dialog') === 'true') {
+        this.skipDialog = true;
+        this.openSnackBar();
```

Comment:
I would just rename the function to something more descriptive. `openUrlSnackbar` or something like